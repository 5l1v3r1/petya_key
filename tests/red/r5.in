88QCVsQJ1v7zXRXhtTSxkC9ZNJi6G4gxikkpNLQZJVAHZHmLdS7LM43zdpUBooz3aRgJVgLR61wqqxvyyQRRhshPMC
