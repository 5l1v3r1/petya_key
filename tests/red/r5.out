msV7BwS7EXWNhaDb
